database/table.go

```diff
@@ -89,7 +90,7 @@ func (t *Table) GetDocument(key []byte) (document.Document, error) {
 	return &d, err
 }
 
-// generate a key for d based on the table configuration.
+// generate a key for d based on the table constraints.
```

Comment:
The table configuration is a generic term that means "based on how the table is configured". "table constraints" however is too specific, as it refers to SQL Table Constraints, which we don't support yet.
Is there a reason you want to change this line?

Reply:
I agree, it's indeed too specific. I'll change it like it was before. 👍